docs/src/pages/components/data-grid/export/export.md

```diff
@@ -8,12 +8,14 @@ title: Data Grid - Export
 
 ## CSV export
 
-You are able to export the displayed data to CSV with an API call, or using the grid UI.
-
-To enable the CSV export you need to compose a toolbar containing the `GridToolbarExport` component, and apply it using the `Toolbar` key in the grid `components` prop.
+The DataGrid allows the data to be exported to CSV by composing a [toolbar](/components/data-grid/components/#toolbar) with the `GridToolbarExport` component. Use the `components` prop to assign the custom toolbar.
 
 {{"demo": "pages/components/data-grid/export/ExportSelectorGrid.js", "bg": "inline"}}
 
+Alternatively, you can export data using the imperative API available in XGrid:
```

Comment:
A detail. I wonder if we shouldn't prefix these sections with apiRef and the pro icon. I wonder because we might be able to:

- avoid frustration for developers using the DataGrid, reading this section only to later realize it's a paid feature.
- create some convention, for instance, we could later use it to identify where we should like the section about how to use the apiRef.

Reply:
If we add the apiRef header in each export output (CSV, Excel, print) we can't have specific links to them, unless we name the sections as "CSV - apiRef". Another alternative would be to have only one "apiRef" at the bottom of the page with the consolidated methods from all outputs, but we don't have a interface that groups them.

Reply:
> we can't have specific links to them

the docs uses a increment `#api-ref`, `#api-ref-2`, etc. I believe.